src/config.ml

```diff
@@ -473,6 +474,17 @@ module Conf_map = struct
        ensure_not Tls_crypt_v2_server
          "server tls-crypt-v2 key passed in tls-crypt-v2")
 
+  let pp_param ppf s =
+    (* XXX(reynir): a_param backslash escaping is probably not correct *)
+    let escape ppf s =
+      Fmt.char ppf '"';
+      Fmt.(list ~sep:(any "\\\"") string) ppf (String.split_on_char '"' s);
+      Fmt.char ppf '"'
+    in
+    if String.exists (function '"' | '\'' | ' ' -> true | _ -> false) s then
+      escape ppf s
+    else Fmt.string ppf s
+
```

Comment:
here I don't escape backslashes as `a_param` seemingly doesn't "unescape" backslashes except for the quote character.

Reply:
To be more precise: when the parser in a quoted string encounters `'\\' '"'` it produces `"\""`. For any other character `c`, `'\\' c` produces `"\\c"`, that is the backslash is kept. So `"\\ "` is parsed as `"\\ "` and not `" "`. That is why I only escape quote characters above.
https://github.com/robur-coop/miragevpn/blob/807ab098c72887939ac036917451f2bd0c9e1c7e/src/config.ml#L886-L891

Reply:
If `\"` is in a string, this will be converted to `\\"`. I'm fine until we hit an issue with that.

Reply:
I expanded on the comment in f2b331bfd68139337f4060f87503c2bf3c900ace. I would revisit this when addressing #257.